### PR TITLE
minor nit

### PR DESCRIPTION
**Pull request by sumith1896**
Ipython -> IPython
@aktech

### CODE REVIEW

**Comment by aktech** [comment]
I think, there is some conflict with my recent push.
Can you fix the merge conflict?

**Comment by sumith1896** [comment]
np, btw the slides link is broken, let's fix that first.

**Comment by aktech** [comment]
Yeah, I just saw that, fixing it.

**Comment by aktech** [comment]
Fixed.

**Comment by sumith1896** [comment]
Cool, let me resolve the merge conflicts.

**Comment by sumith1896** [comment]
Done @aktech

File: index.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
            href="#" >Tutorial</a>
             <ul>
             <li><a title='Intro slides' href="http://iamit.in/sympy-pycon/pycon2015.pdf" target="_blank" rel=external>Slides</a></li>
-            <li><a title='Ipython Notebooks' href="http://nbviewer.ipython.org/github/aktech/sympy-pycon/tree/master/PyCon%20material/tutorial_exercises/" target="_blank" rel=external>Ipython Notebooks</a></li>
+            <li><a title='Ipython Notebooks' href="http://nbviewer.ipython.org/github/aktech/sympy-pycon/tree/master/PyCon%20material/tutorial_exercises/" target="_blank" rel=external>IPython Notebooks</a></li>
```
Comment (aktech) [inline index.html]: Please change this also `title='Ipython Notebooks'`.

**Comment by aktech** [comment]
Thanks! @Sumith1896

**Comment by sumith1896** [comment]
Thanks @aktech

**Pull request merged**